commit d7b6dc3982b31aeee04237ac5735e905d95d5510
Author: Wei Yongjun <weiyongjun1@huawei.com>
Date:   Tue Feb 7 15:51:47 2017 +0000

    tpm_tis: fix the error handling of init_tis()
    
    commit 5939eaf4f9d432586dd2cdeea778506471e8088e upstream.
    
    Add the missing platform_driver_unregister() and remove the duplicate
    platform_device_unregister(force_pdev) in the error handling case.
    
    Fixes: 00194826e6be ("tpm_tis: Clean up the force=1 module parameter")
    Signed-off-by: Wei Yongjun <weiyongjun1@huawei.com>
    Reviewed-by: Jason Gunthorpe <jgunthorpe@obsidianresearch.com>
    Reviewed-by: Jarkko Sakkinen <jarkko.sakkinen@linux.intel.com>
    Signed-off-by: Jarkko Sakkinen <jarkko.sakkinen@linux.intel.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1246_d7b6dc3982b31aeee04237ac5735e905d95d5510.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1246_d7b6dc3982b31aeee04237ac5735e905d95d5510.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
